mhs/common/mhs_common/errors/ebxml_handler.py

```diff
@@ -60,16 +59,6 @@ def handle_ebxml_error(code: int, headers: Dict, body: str) -> Tuple[int, Option
     :param body: HTTP response body
     :return: Response to external client represented as HTTP status code and body
     """
-
-    if code != 200:
-        if code // 100 == 2:  # If code is 2xx
-            logger.info('0001', "HTTP {Code} success response code received. So can assume this isn't an ebXML error "
-                                "as they are HTTP 200.", {'Code': code})
-            return code, body
-        logger.error('0002', 'Non-HTTP 2xx response code received: {Code} . It is likely that there is a bug in the '
-                             'code.', {'Code': code})
-        raise ValueError(f'Non-HTTP-2xx code passed to ebxml_handler function: {code}')
-
```

Comment:
I discussed this with Colin, we concluded that the ebxml error parser shouldn't care about HTTP status codes, removing this line ensures the functionality is correct but I'll also add a tech debt ticket describing that really this class should have all references to HTTP codes removed